package/access.logrotate

```diff
@@ -1,5 +1,5 @@
 compress
-/var/log/yig/access.log {
+/var/log/yig/access.log* {
```

Comment:
May re-rotate the log files that have already been rotated?

Reply:
maybe...